### ISSUE TITLE
datalaoder issues

### ISSUE THREAD

**Issue by whuhangzhang** [WenmuZhou/PytorchOCR#72]
When loading data, the length of the data and the characters not in the character set should be filtered, some complex mistakes can be made in the course of the experiment.
- When the length of the input label is zero, the error of divisor 0 will appear in the calculation of editing distance at the beginning of training

https://github.com/WenmuZhou/PytorchOCR/blob/866651e6baa360f191d3560f44d3bfe9f3134f1b/tools/rec_train.py#L234
https://github.com/WenmuZhou/PytorchOCR/blob/866651e6baa360f191d3560f44d3bfe9f3134f1b/torchocr/metrics/RecMetric.py#L22-L24

- If the length exceeds the set length, it also needs to be filtered. At present, I did not find relevant Settings in config file

https://github.com/WenmuZhou/PytorchOCR/blob/5028c9db1f2f3efea70188d27b0adcfacb89565e/torchocr/datasets/RecDataSet.py#L90-L95
https://github.com/WenmuZhou/PytorchOCR/blob/866651e6baa360f191d3560f44d3bfe9f3134f1b/config/rec_train_lmdb_config.py#L73-L80

**Comment by WenmuZhou** [comment]
I will add filter code

**Comment by whuhangzhang** [comment]
thank you ~
